Can't install the API server :: Cannot find module '../build/Release/sharp.node'

Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)


## Issue Description

I can't install the API server.
Here is the log:

```
# docker-compose logs -f api
Attaching to reaction_api_1
api_1    |
api_1    | > reaction-api@3.13.0 start
api_1    | > node --experimental-modules --experimental-json-modules ./src/index.js
api_1    |
api_1    | (node:19) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
api_1    | (Use `node --trace-warnings ...` to show where the warning was created)
api_1    | 03:47:26.814Z ERROR Reaction:
api_1    |
api_1    |   Something went wrong installing the "sharp" module
api_1    |
api_1    |   Cannot find module '../build/Release/sharp.node'
api_1    |   Require stack:
api_1    |   - /usr/local/src/app/node_modules/@reactioncommerce/api-plugin-files/node_modules/sharp/lib/constructor.js
api_1    |   - /usr/local/src/app/node_modules/@reactioncommerce/api-plugin-files/node_modules/sharp/lib/index.js
api_1    |
api_1    |   - Run "npm rebuild --verbose sharp" and look for errors
api_1    |   - Consult the installation documentation at https://sharp.pixelplumbing.com/install
api_1    |   - Search for this error at https://github.com/lovell/sharp/issues
api_1    |
api_1    |   --
api_1    |   Error:
api_1    |   Something went wrong installing the "sharp" module
api_1    |
api_1    |   Cannot find module '../build/Release/sharp.node'
api_1    |   Require stack:
api_1    |   - /usr/local/src/app/node_modules/@reactioncommerce/api-plugin-files/node_modules/sharp/lib/constructor.js
api_1    |   - /usr/local/src/app/node_modules/@reactioncommerce/api-plugin-files/node_modules/sharp/lib/index.js
api_1    |
api_1    |   - Run "npm rebuild --verbose sharp" and look for errors
api_1    |   - Consult the installation documentation at https://sharp.pixelplumbing.com/install
api_1    |   - Search for this error at https://github.com/lovell/sharp/issues
api_1    |
api_1    |       at Object.<anonymous> (/usr/local/src/app/node_modules/@reactioncommerce/api-plugin-files/node_modules/sharp/lib/constructor.js:34:9)
api_1    |       at Module._compile (internal/modules/cjs/loader.js:1063:30)
api_1    |       at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
api_1    |       at Module.load (internal/modules/cjs/loader.js:928:32)
api_1    |       at Function.Module._load (internal/modules/cjs/loader.js:769:14)
api_1    |       at Module.require (internal/modules/cjs/loader.js:952:19)
api_1    |       at require (internal/modules/cjs/helpers.js:88:18)
api_1    |       at Object.<anonymous> (/usr/local/src/app/node_modules/@reactioncommerce/api-plugin-files/node_modules/sharp/lib/index.js:3:15)
api_1    |       at Module._compile (internal/modules/cjs/loader.js:1063:30)
api_1    |       at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
api_1    | npm notice
api_1    | npm notice New minor version of npm available! 7.5.4 -> 7.20.3
api_1    | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.20.3>
api_1    | npm notice Run `npm install -g npm@7.20.3` to update!
api_1    | npm notice
api_1    | npm ERR! code 1
api_1    | npm ERR! path /usr/local/src/app
api_1    | npm ERR! command failed
api_1    | npm ERR! command sh -c node --experimental-modules --experimental-json-modules ./src/index.js
api_1    |
api_1    | npm ERR! A complete log of this run can be found in:
api_1    | npm ERR!     /home/node/.npm/_logs/2021-07-30T03_47_26_832Z-debug.log
api_1    | reaction_api_1 exited with code 1
```



### Versions

```
# npm run version --silent
3.13.1
```

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "@reactioncommerce/reaction-error": "~1.0.1",
     "envalid": "~5.0.0",
     "node-fetch": "~2.6.0",
-    "sharp": "~0.26.1",
+    "sharp": "^0.28.3",
     "simpl-schema": "~1.10.2"
   },
   "devDependencies": {

